<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEncWindow</class>
 <widget class="QMainWindow" name="FileEncWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>program_title</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon.png</normaloff>icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>program_title</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="file_enc_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>227</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>File Encryption</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_select_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>88</y>
      <width>240</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Drag or drop file here.</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Drag &amp; Drop file here</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="file_select_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>88</y>
      <width>30</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Select file</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="pw_entry">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>139</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Enter the password you want to set for the file.</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter file password</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="pw_cf_entry">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>168</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Please re-enter the password you want to use for the file.</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter file password (confirm)</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>240</y>
      <width>278</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_box">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>280</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="toolTip">
     <string>If you forget your password, you won't be able to recover your files</string>
    </property>
    <property name="text">
     <string>I remembered my file password*          </string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>return to menu</string>
    </property>
    <property name="text">
     <string>&lt; Back</string>
    </property>
   </widget>
   <zorder>progress_bar</zorder>
   <zorder>title_label</zorder>
   <zorder>file_enc_btn</zorder>
   <zorder>file_select_label</zorder>
   <zorder>file_select_btn</zorder>
   <zorder>pw_entry</zorder>
   <zorder>pw_cf_entry</zorder>
   <zorder>check_box</zorder>
   <zorder>back_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
